apps/accounts/tests/test_views.py

```diff
@@ -119,7 +123,7 @@ def test_profile_exists_with_user(self):
         self.fxa_identify.return_value = identity
         self.find_user.return_value = self.user
         self.user.is_authenticated.return_value = False
-        self.request.DATA = {'code': 'foo'}
+        self.request.DATA = {'code': 'foo', 'state': 'Alabama'}
```

Comment:
using states not states is funny but I think someone will be confused. It would be more boring but something like 'fxa-csrf-state' would make the test easier to follow for someone new to the code. `</DebbieDowner>`